frontend/src/shared/pages/Home.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <HelloWorld />
```

Comment:
remove HelloWorld?

Reply:
yeah - why not?
Replaced it with Splashscreeb